packages/paste-core/components/alert/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -77,8 +77,8 @@ exports[`Alert Variant error Should render an error alert 1`] = `
       >
         <span
           className="emotion-0"
-          color="colorTextError"
-          data-paste-element="ICON"
+          color="colorTextIconError"
+          data-paste-element="BOX"
```

Comment:
Did you install and build after rebasing with main? I saw this snapshot change from `ICON` to `BOX` too before I installed the most up to date package versions.

Reply:
I did not install after rebasing! Will do 👍 

I updated snapshots so those should be good next time I push up